packages/mermaid/src/diagrams/sequence/svgDraw.js

```diff
@@ -343,10 +343,10 @@ const drawActorTypeParticipant = async function (elem, actor, conf, isFooter) {
       .attr('y1', centerY)
       .attr('x2', center)
       .attr('y2', 2000)
-      .attr('class', 'actor-line')
-      .attr('class', '200')
+      .attr('class', 'actor-line 200')
```

Comment:
I  feel the one in this PR looks nicer since the line color is more prominent :) 
However, I am happy to keep it as the one in production as well.  Let me know which way we want to go ahead with and based on that I can push the changes as needed.
cc @aloisklink @sidharthv96

Reply:
friendly reminder @aloisklink @sidharthv96 :)

Reply:
> However, I am happy to keep it as the one in production as well.

It might be better to keep the color as it is, even if it does look slightly worse, since this bug has been around so long that people are probably used to the current colour.

Plus, due to the way this repo has GitHub permissions set up, if the visual regression tests show changes, we need an @mermaid-js admin to merge this PR. Otherwise, I can merge it myself :smile:

Reply:
> It might be better to keep the color as it is, even if it does look slightly worse, since this bug has been around so long that people are probably used to the current colour.

Please let's not make this a reason for not pushing better changes to the UI when we know agree its visually a better one :) It will be a change but visually better one so users will gradually start liking it :)

Reply:
@ad1992, for this PR, do you want to apply the above diff (either manually or by copying to a `.diff` file and running `git apply my-diff-file.diff`?) to prevent any visual changes? If there are no visual changes for this PR, I can merge it without waiting for anybody else.

It's probably easier to make the visual changes in another PR, since that will require admin permissions from another reviewer. I'd imagine we'd probably also want to pick a setting that looks good in all of the [built-in mermaid themes](https://mermaid.js.org/config/theming.html#available-themes), e.g.

<table>
  <tr>
    <td> default </td>
    <td> neutral </td>
    <td> dark </td>
    <td> forest </td>
    <td> base </td>
  </tr>
  <tr>
  <td>

```mermaid
---
config:
  theme: default
---
sequenceDiagram
    Alice->>+John: Hello John, how are you?
```
  </td>
  <td>

```mermaid
---
config:
  theme: neutral
---
sequenceDiagram
    Alice->>+John: Hello John, how are you?
```
  </td>
  <td>

```mermaid
---
config:
  theme: dark
---
sequenceDiagram
    Alice->>+John: Hello John, how are you?
```
  </td>
  <td>

```mermaid
---
config:
  theme: forest
---
sequenceDiagram
    Alice->>+John: Hello John, how are you?
```
  </td>
  <td>

```mermaid
---
config:
  theme: base
---
sequenceDiagram
    Alice->>+John: Hello John, how are you?
```
  </td>
  </tr>
</table>